examples/pytorch/infograph/qm9_v2.py

```diff
@@ -0,0 +1,213 @@
+import numpy as np
```

Comment:
You may add this class to DGL core as other datasets.

Reply:
I will create another PR for it.

Reply:
I guess we should merge the PR for the dataset first as this PR depends on that.